### PR TITLE
update pre_split_regions usage scenario

### PR DESCRIPTION
**Pull request by pepezzzz**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Clarify that PRE_SPLIT_REGIONS is also recommended for tables with AUTO_RANDOM attribute

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.3 (TiDB 8.3 versions)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v8.0 (TiDB 8.0 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by bb7133 — APPROVED** [review]
LGTM

File: sql-statements/sql-statement-split-region.md [hunk 1]
```diff
@@ -351,11 +351,11 @@ region4  [("c", "")                    , maxIndexValue               )
 
 ## pre_split_regions
 
-使用带有 `SHARD_ROW_ID_BITS` 的表时，如果希望建表时就均匀切分 Region，可以考虑配合 `PRE_SPLIT_REGIONS` 一起使用，用来在建表成功后就开始预均匀切分 `2^(PRE_SPLIT_REGIONS)` 个 Region。
+使用带有 `AUTO_RANDOM` 和 `SHARD_ROW_ID_BITS` 关键字的表时，如果希望建表时就均匀切分 Region，可以考虑配合 `PRE_SPLIT_REGIONS` 一起使用，用来在建表成功后就开始预均匀切分 `2^(PRE_SPLIT_REGIONS)` 个 Region。
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-split-region.md:354]:
```suggestion
使用带有 `AUTO_RANDOM` 或 `SHARD_ROW_ID_BITS` 属性的表时，如果希望建表时就均匀切分 Region，可以考虑配合 `PRE_SPLIT_REGIONS` 一起使用，用来在建表成功后就开始预均匀切分 `2^(PRE_SPLIT_REGIONS)` 个 Region。
```

File: sql-statements/sql-statement-split-region.md [hunk 2]
```diff
@@ -351,11 +351,11 @@ region4  [("c", "")                    , maxIndexValue               )
...
 
 > **注意：**
 >
-> `PRE_SPLIT_REGIONS` 必须小于等于 `SHARD_ROW_ID_BITS`。
+> `PRE_SPLIT_REGIONS` 的值必须小于或等于 `SHARD_ROW_ID_BITS` 和 `AUTO_RANDOM` 的值。
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-split-region.md:358]:
```suggestion
> `PRE_SPLIT_REGIONS` 的值必须小于或等于 `SHARD_ROW_ID_BITS` 或 `AUTO_RANDOM` 的值。
```

**Review by Oreoxmt — APPROVED** [review]

File: sql-statements/sql-statement-split-region.md [hunk 3]
```diff
@@ -351,11 +351,11 @@ region4  [("c", "")                    , maxIndexValue               )
 
 ## pre_split_regions
 
-使用带有 `SHARD_ROW_ID_BITS` 的表时，如果希望建表时就均匀切分 Region，可以考虑配合 `PRE_SPLIT_REGIONS` 一起使用，用来在建表成功后就开始预均匀切分 `2^(PRE_SPLIT_REGIONS)` 个 Region。
+使用带有 `AUTO_RANDOM` 或 `SHARD_ROW_ID_BITS` 属性的表时，如果希望建表时就均匀切分 Region，可以考虑配合 `PRE_SPLIT_REGIONS` 一起使用，用来在建表成功后就开始预均匀切分 `2^(PRE_SPLIT_REGIONS)` 个 Region。
```
Comment (qiancai) [inline sql-statements/sql-statement-split-region.md:354]:
```suggestion
创建带有 `AUTO_RANDOM` 或 `SHARD_ROW_ID_BITS` 属性的表时，如果希望在建表成功后就开始预均匀切分 Region，可以考虑配合 `PRE_SPLIT_REGIONS` 一起使用。预切分成的 Region 数量为 `2^(PRE_SPLIT_REGIONS)`。
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Pull request merged**